src/signals/incident-management/containers/IncidentOverviewPage/contants.ts

```diff
@@ -50,16 +52,24 @@ export type SortOption = {
 
 export const sortOptionsList: SortOption[] = [
   {
-    key: SortOptionKeys.ADRES,
-    label: SortOptionLabels.ADRES,
+    key: SortOptionKeys.ADDRESS,
+    label: SortOptionLabels.ADDRESS,
+    asc: SortOptions.ASSIGNED_USER_EMAIL_ASC,
```

Comment:
This seems to be swapped. Shouldn't the key `SortOptionKeys.ADDRESS` have `SortOptions.ADDRESS_ASC`?